src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -591,18 +698,22 @@
    environment-class-cache-enabled :- schema/Bool]
   (let [environment-class-handler
         (environment-class-handler jruby-service
-                                   environment-class-cache-enabled)
+                                   environment-class-cache-enabled),
         environment-module-handler
-        (environment-module-handler jruby-service)
+        (environment-module-handler jruby-service),
         all-tasks-handler
-        (all-tasks-handler jruby-service)
+        (all-tasks-handler jruby-service),
+        task-details-handler
+        (task-details-handler jruby-service),
```

Comment:
this is very much an opinion thing but I hate commas and prefer newlines if necessary, but I still find this readable without commas (and maybe the two-line situation isn't necessary?)

Reply:
i also find the commas off-putting

Reply:
murdering the commas, replacing with newlines